### PR TITLE
UI update

### LINKED ISSUE #75: Improve UI of solution Upload page
This is task is simple and is to make UI better for the upload page.
You can use CSS Libraries unless they are too heavy.

To be done after #73

### PR DESCRIPTION
**Pull request by mark-nakachon**
### Checklist:

* [x] I followed the guidelines in Contributing document
* [x] I checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change
* [x] I successfully ran tests with my changes locally?
* [x] I updated the UI for solution file upload

### Issues Solved

Fixes #75

### CODE REVIEW

**Comment by mark-nakachon** [comment]
i updated the UI

**Comment by mark-nakachon** [comment]
i resolved the conflicts

**Comment by mark-nakachon** [comment]
I made the changes you requested please merge it now

File: server/index.html [hunk 1]
```diff
@@ -1,17 +1,65 @@
...
-        <h1>Upload File Here</h1>
-        <input type="file" name="upfile" value="">
-        <input type="submit" />
+<style>
+    input[type="file"]{
+        display: none;
+    }
+</style>
+<body class="bg-dark">
+    <form class="fileupload" action="upload" method="post" enctype="multipart/form-data">
```
[the remark this answers is not in the document]
Comment (aashutoshrathi) [reply server/index.html]: @mark-nakachon Please look at reviews carefully.
It won't work without you committing this change

**Comment by mark-nakachon** [comment]
changed the action attribute and I will  contribute more  to this project, open the issues which u require I will try to fix them and don't forget to add the gssoc label

**Pull request merged**